src/dc_receive_imf.rs

```diff
@@ -652,6 +652,17 @@ unsafe fn add_parts(
                             .set_int(Param::Cmd, mime_parser.is_system_message as i32);
                     }
 
+                    /*
+                    info!(
+                        context,
+                        "received mime message {:?}",
+                        String::from_utf8_lossy(std::slice::from_raw_parts(
+                            imf_raw_not_terminated as *const u8,
+                            imf_raw_bytes,
+                        ))
+                    );
+                    */
```

Comment:
cand this be deleted?

Reply:
i often need it -- and also for the pending debugging of incoming messages ending up as "[]" in the db.